core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -860,26 +870,37 @@ public void run() {
                                 .setContentIntent(pIntent)
```

Comment:
I can see how that works. I was worried it wouldn't accomplish everything the stop button (X) does.  But maybe people don't care about all that stuff...